_posts/2019-09-10-PC-Engines-Firmware-v4-10-0-1.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.10.0.1"
+date:   2019-09-10
+categories: Firmware
+---
+# PC Engines apu coreboot Open Source Firmware v4.10.0.1
+
+## Key changes
+
+Mainline:
+
+1. **Rebased** with official coreboot repository **commit** 22d66ef.
+2. **Enabled watchdog** on **apu3**.
+
+Legacy:
+1. **Enabled watchdog** on **apu3**.
+
```

Comment:
@piotr-kleins there is one patch merged https://review.coreboot.org/c/coreboot/+/35286